debug.log

```diff
@@ -0,0 +1 @@
+[0820/113142.843:ERROR:icu_util.cc(252)] Couldn't mmap icu data file
```

Comment:
What is the need of this file?

Reply:
This one is from the vscode

Reply:
don't commit any unnecessary file, please remove it.